### PR TITLE
Add test for node adaptation on Squeeze11

### PR DESCRIPTION
**Pull request by aivanoved**
This test shows that simple subgraph adaptation works

In contrast to #145 

# Checklist

- [x] ~~Added a `CHANGELOG.rst` entry~~
Note: No changes, which necessitate an addition to the changelog

### CODE REVIEW

File: tests/test_adapt.py [hunk 1]
```diff
@@ -233,3 +236,30 @@ def test_inline_model_custom_node_nested(old_squeeze: onnx.ModelProto):
...
+        None, {"b": b, "cond": cond}
+    )
+
+
+def test_if_adaptation_const():
+    sq = op19.const(1.1453, dtype=numpy.float32)
+    b = argument(Tensor(numpy.float32, ("N",)))
+    cond = op18.equal(sq, b)
+    out = op18.if_(cond, then_branch=lambda: [sq], else_branch=lambda: [sq])
+    build({"b": b}, {"out": out[0]})
```
Comment (cbourjau) [inline tests/test_adapt.py:265]: Can you check the opset version of the `"ai.onnx"` aka `""` of the built artifact please? It should be larger than 11.

**Review by cbourjau — CHANGES_REQUESTED** [review]
Is something missing to push this over the finish line?

File: tests/test_adapt.py (hunk 1 above)
Comment (aivanoved) [reply tests/test_adapt.py:265]: Yes, can do

**Comment by aivanoved** [comment]
Don't believe so

**Review by cbourjau — CHANGES_REQUESTED** [review]
Open comment is still relevant.

**Review by cbourjau — CHANGES_REQUESTED** [review]
Thanks for getting back to this. One last minor comment.

**Comment by aivanoved** [comment]
@cbourjau

I don't see the comment

File: tests/test_adapt.py [hunk 2]
```diff
@@ -71,32 +72,34 @@ def inline_old_identity_twice_graph(old_identity):
...
 
-        attrs: Attributes
-        inputs: Inputs
-        outputs: Outputs
+    attrs: Attributes
+    inputs: Inputs
+    outputs: Outputs
 
-    def squeeze11(_data: Var, _axes: Iterable[int]):
+    @classmethod
```
Comment (cbourjau) [inline tests/test_adapt.py:94]: Why does this need to be a class method? Usually our constructor functions are free functions.

Comment (aivanoved) [inline tests/test_adapt.py:94]:
```suggestion
```

File: tests/test_adapt.py [hunk 3]
```diff
@@ -71,32 +72,33 @@ def inline_old_identity_twice_graph(old_identity):
...
 
-        attrs: Attributes
-        inputs: Inputs
-        outputs: Outputs
+    attrs: Attributes
+    inputs: Inputs
+    outputs: Outputs
 
-    def squeeze11(_data: Var, _axes: Iterable[int]):
+    def squeeze11(cls, _data: Var, _axes: Iterable[int]):
```
Comment (aivanoved) [inline tests/test_adapt.py:94]:
```suggestion
    def squeeze11(self, _data: Var, _axes: Iterable[int]):
```

**Review by cbourjau — APPROVED** [review]

**Pull request merged**